ceph-releases/ALL/centos/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -49,6 +49,8 @@ bash -c ' \
       ARCH="arm64"; \
     fi ; \
     REPO_URL=$(curl -s "https://shaman.ceph.com/api/search/?project=ceph&distros=centos/__ENV_[BASEOS_TAG]__/${ARCH}&flavor=${OSD_FLAVOR}&ref=${CEPH_REF}&sha1=latest" | jq -r .[0].url); \
+    yum install -y dnf-plugins-core ; \
+    yum copr enable -y tchaikov/python-scikit-learn ; \
```

Comment:
does this actually get the package installed?

Reply:
No this only enables the copr repository so we need an explicit `yum install -y python3-scikit-learn` command as well

Reply:
no, it does not. i assumed that the package list was located somewhere else.

Reply:
@liewegas @dsavineau @yaarith  fixed and repushed.